streamly.cabal

```diff
@@ -202,7 +202,7 @@ extra-source-files:
     core/src/Streamly/Internal/Data/Stream/Zip.hs
     core/src/Streamly/Internal/Data/List.hs
     core/src/Streamly/Internal/Data/Array.hs
-    core/src/Streamly/Internal/Data/Array/Foreign.hs
+    core/src/Streamly/Internal/Data/Array/*.hs
```

Comment:
Better not use `*` as it might pick up any stale, backup files lying around in the workspace.